db/migrate/20200614010627_update_item_value.rb

```diff
@@ -0,0 +1,18 @@
+class UpdateItemValue < ActiveRecord::Migration[5.0]
+  def change
+    change_column :items, :value, :decimal, default: 0.01, null: false
+
+    nil_value_items  = Item.where(value: nil)
+    zero_value_items = Item.where("value < 0.01")
+
+    if nil_value_items.present?
+      puts "Found `#{nil_value_items.count}` items with a value of nil, updating to `0.01`."
+      nil_value_items.update_all(value: 0.01)
```

Comment:
Do we care about updating `updated_at`? This does't do that automatically.

Reply:
oh right, no callbacks, prob doesn't matter but adding so it's in papertrail